kernel/src/udp_port_table.rs

```diff
@@ -0,0 +1,249 @@
+//! UDP port table implementation for enforcing port binding. This table is
```

Comment:
Blank line after a short header.

Also I think creating a `net` folder to mirror the capsules would be useful.